docs/api/csf.md

```diff
@@ -238,3 +253,154 @@ For this particular example, you could achieve the same result in different ways
 - `excludeStories: ['simpleData', 'complexData']`
 
 The first option is the recommended solution if you follow the best practice of starting story exports with an uppercase letter (i.e., use UpperCamelCase).
+
+## Upgrading from CSF 2 to CSF 3
```

Comment:
I could not determine a suitable spot in the TOC, so I went with a new section on this page. But if someone thinks it deserves its own page, I'm open to suggestions on where it would be placed.

Reply:
As discussed elsewhere, rather than move or duplicate the section, I added a link to it.